tests/framework/extensions/machinepools/machinepools.go

```diff
@@ -0,0 +1,67 @@
+package machinepools
```

Comment:
helper to actually do the create

`func CreateMachinePools(rancherClient *rancher.Client, ...) MachinePool, error`